src/Handler/Workspace/Create/CreateWorkspaceHandler.php

```diff
@@ -142,18 +144,32 @@ public function __invoke(
 
         $proxy->call(function () use ($cloudResourceManager, $projectName, $wsServiceAcc): void {
             $getIamPolicyRequest = new GetIamPolicyRequest();
-            $actualPolicy = $cloudResourceManager->projects->getIamPolicy($projectName, $getIamPolicyRequest, []);
+            $option = new GetPolicyOptions();
+            $option->setRequestedPolicyVersion(Helper::REQUESTED_POLICY_VERSION);
+            $getIamPolicyRequest->setOptions($option);
+            $actualPolicy = $cloudResourceManager->projects->getIamPolicy($projectName, $getIamPolicyRequest);
             $finalBinding[] = $actualPolicy->getBindings();
 
             foreach (self::IAM_WORKSPACE_SERVICE_ACCOUNT_ROLES as $role) {
                 $bigQueryJobUserBinding = new Binding();
                 $bigQueryJobUserBinding->setMembers('serviceAccount:' . $wsServiceAcc->getEmail());
                 $bigQueryJobUserBinding->setRole($role);
+
+                if ($role === IAmPermissions::ROLES_BIGQUERY_DATA_VIEWER) {
```

Comment:
Chcem to nastavit len BQ data viewer roli ktora sluzi nieco ako RO, takze chceme RO iba na datasety bucketov nie WS